uimaj-parent/pom.xml

```diff
@@ -472,6 +455,28 @@
                 </goals>
               </execution>
             </executions>
+            <dependencies>
+              <dependency>

```

Comment:
any way to factor this common set of dependencies into one spot?

Reply:
As UIMA dependencies, we can add them to the parent POM such that all modules inherit them.

However, if individual Maven plugins require these dependencies (such as here), I know of no way to do a single declaration that would affect all plugins. Mind that that each plugin has its own classloader.

Reply:
ok